poetry.lock

```diff
@@ -14,6 +22,40 @@ optional = false
 python-versions = "*"
 version = "0.1.0"
 
+[[package]]
+category = "dev"
+description = "Classes Without Boilerplate"
+name = "attrs"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+version = "19.3.0"
+
+[package.extras]
+azure-pipelines = ["coverage", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "zope.interface", "pytest-azurepipelines"]
```

Comment:
Do we need all the stuff in the lock file?

Reply:
Yes otherwise people do not use the same packages.